source/_components/ads.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: page
+title: "ADS"
+description: "Connect Home Assistant to TwinCAT devices via the ADS interface."
+date: 2016-05-25 23:39
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: Hub
+ha_release: 0.59
+ha_iot_class: "Local Push"
+---
+
+The ADS (automation device specification) describes a device-independent and
+fieldbus-independent interface for communication between Beckhoff® automation
+devices running TwinCAT® and other devices implementing this interface.
+
+Configuration parameters:
+
+{% configuration %}
+
+- **device** (*Required*): The AMS NetId that identifies the device.
```

Comment:
This should be written using a kind of yaml syntax. See:
https://raw.githubusercontent.com/home-assistant/home-assistant.github.io/next/source/_components/dominos.markdown

Reply:
Done.